components/docusign/actions/create-signature-request/create-signature-request.js

```diff
@@ -0,0 +1,78 @@
+const docusign = require("../../docusign.app.js");
+
+module.exports = {
+  key: "docusign-create-signature-request",
+  name: "Create Signature Request",
+  description: "Creates a signature request from a template",
+  version: "0.0.01",
```

Comment:
You have an extra "0" in front of the last "1" :)

Reply:
Updated